### DISCUSSION TITLE
弹幕点赞功能还有支持计划吗？

### DISCUSSION THREAD

**Discussion by timongh** [the1812/Bilibili-Evolved#3161] [综合讨论 / General]
我一段时间前研究过这个问题，发现是可以解决的。我还写了一个文档，大体解决方案也写里面了，只是没写代码而已。

所以想问问，目前这个问题是永远不支持了，还是怕 B 站界面不稳定，才延缓支持。如果还在支持计划里的话，我把我写的文档润色润色发出来。具体代码谁有兴趣可以认领，或者过段时间我写也行（不确定，总之先挖个坑）

**Comment by the1812** [comment]
目前不在支持计划里, 除非有其他重要功能也被塞到弹幕菜单里了

**Reply by timongh** [reply]
好吧，那还挺遗憾的 [摊手]

**Comment by Geeyun-JY3** [comment]
之前倒是有个相关的需求：#2832
